pkg/apis/operator/v1alpha1/operandconfig_types.go

```diff
@@ -106,7 +105,7 @@ func init() {
 //InitConfigStatus OperandConfig status
 func (r *OperandConfig) InitConfigStatus() {
 	if (reflect.DeepEqual(r.Status, OperandConfigStatus{})) {
-		r.Status.Phase = ServicePending
+		r.Status.Phase = ServiceReady
```

Comment:
Why InitConfigStatus is ready?

Reply:
I updated the OperandConfig and OpernadRegistry init status to `pending` yesterday. it will be updated to `ready` after OperanRequest CR is generated. However, some users report bugs when they see `pending` status in the `OperandConfig` and `OpernadRegistry` CR.

Reply:
Can we add a state named as `Init`? This maybe more clear to end users.

Reply:
Sure, will do

Reply:
@gyliu513 Done, using `Initialized` as the init status.